trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'name_decorator'
+
+class TrimmerDecorator < NameDecorator
+  def correct_name
+    @nameable.correct_name[0..10]
```

Comment:
- [ ] This method returns 11 characters ("Maximilianu"), instead of 10 characters ("Maximilian"). Kindly fix that by adding an extra dot.

  ```
  @nameable.correct_name[0...10]
  ```

Reply:
Thank you @TedLivist I'll fix it